run_flake8.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -euo pipefail
+
+directory=${1:-.}
+flake8 --exclude thirdparty,build --ignore E501,W503 --per-file-ignores="steps.py:F811" "${directory}"
```

Comment:
If we add `--builtins log,REL_SUCCESS,REL_FAILURE` here, then we can get rid of most of the `# noqa: F821` comments.

Reply:
Good idea! Updated in 51b7f780e0497c5695321631e9c7bd0f655ddcc1